src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/build/Microsoft.DotNet.UpgradeAssistant.Abstractions.targets

```diff
@@ -0,0 +1,204 @@
+﻿<Project>
+
+  <!--
+    This is a list of all the host-supplied files that should not be contained within an extension.
+    
+    To create a new list, go to the output of the CLI and run the following sommand in PowerShell:
+    
+    Get-ChildItem *.dll -Recurse -Exclude extensions `
+      |% { $_.FullName.Replace("$pwd\", "") } `
+      |% { "<_UpgradeExtensionExclude Include=`"$_`" />" } `
+      | Set-Clipboard
+     
+    An updated list will now be on the clipboard and can be pasted below.
```

Comment:
this looks more like part of our dev loop process than an update to a doc

Reply:
Additions of the target files need to be called out in the doc?

Reply:
Shouldn't matter to devs - it just ensures that their builds produce useful results